docs/guides/external-node/02_configuration.md

```diff
@@ -5,6 +5,9 @@ environment variables. To streamline this process, we provide prepared configs f
 [mainnet](prepared_configs/mainnet-config.env) and [testnet](prepared_configs/testnet-sepolia-config.env). You can use
 these files as a starting point and modify only the necessary sections.
 
+**You can also see directory docker-compose-examples if you want to run external-node on your machine with recommended
```

Comment:
Nit: You can like to the directory, and also use a `[!NOTE]` for better structuring. Ditto for the note in *Running the External Node*.

Reply:
corrected